js/components/dashboard/generic-answer.js

```diff
@@ -0,0 +1,17 @@
+import React, { PureComponent } from 'react'
+
+import css from '../../../css/dashboard/generic-answer.less'
+
+export default class GenericAnswerCell extends PureComponent {
+  render () {
+    const { answer } = this.props
```

Comment:
I didn't end up needing an `expanded` prop since rendering the icon was so simple, unless I'm missing something.